frontend/test/metabase/scenarios/visualizations/trendline.cy.spec.js

```diff
@@ -27,12 +27,7 @@ describe("scenarios > question > trendline", () => {
   it("displays trendline when there are multiple numeric outputs (for simple question) (metabase#12781)", () => {
     // Change settings to trendline
     cy.findByText("Visualization").click();
-    sidebar().within(() => {
-      cy.icon("line").click();
-    });
-    cy.findByTestId("Line-button").within(() => {
-      cy.icon("gear").click();
-    });
+    cy.findByTestId("viz-settings-button").click();
```

Comment:
Simplifying this test since we specify `display:"line"` in the Question and we can get straight to the settings panel through the `viz-settings-button`.